lib/stm32wb/hci/README

```diff
@@ -61,15 +61,8 @@ License Link:
 
 Patch List:
 
-        * Move some files license from ST SLA0044 to BSD-3 Clause
-            Following files are distributed under ST License SLA0044.
-            For Zephyr they are made available under BSD-3 Clause.
-            - hw_if.h
-            - hw_ipcc.c
-            - app_common.h
-            - app_conf.h
-            - stm32_wpan_common.h
-            - ble_bufsize.h
+        * all files are now are distributed under BSD-3 Clause.
+            which terms are available at hal/stm32 directory
```

Comment:
Headers explicitely refers to "the root directory of this software component".
We need to comply with that, so please add a BSD-3 Clause license file, along this change, under lib/stm32wb/hci.

Reply:
[LICENSE](https://github.com/zephyrproject-rtos/hal_stm32/blob/main/LICENSE) file is available for 
lib/stm32wb/hci and stm32cube/stm32XXxx